### PR TITLE
Add commands for running as a standalone app

### PR DESCRIPTION
**Pull request by jzoldak**
@peter-fogg @clintonb Would you guys be able to either review this for me, or suggest someone else who could?

### CODE REVIEW

File: course_discovery/urls_local.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""
```
Comment (clintonb) [inline course_discovery/urls_local.py]: Why is this file necessary? If you want a `user_info` endpoint, add it behind a setting in the normal `urls.py`.

**Comment by peter-fogg** [comment]
@jzoldak What's the purpose of this work? Is it to stub out course discovery for use in integration/acceptance tests?

**Comment by jzoldak** [comment]
@peter-fogg It's so that you can run acceptance tests locally against a running instance. The idea is that the same automated acceptance tests can be pointed either locally or against stage, etc. They will use different data and possibly a different set of tests (e.g. query-only tests vs. ones that might add/delete data) depending on the target environment.

File: course_discovery/settings/utils.py [hunk 2]
```diff
@@ -10,3 +11,15 @@ def get_env_setting(setting):
     except KeyError:
         error_msg = "Set the [{}] env variable!".format(setting)
         raise ImproperlyConfigured(error_msg)
+
+
+def stub_user_info_response(request):
+    """ Stub the user info endpoint response of the OAuth2 provider. """
```
Comment (clintonb) [inline course_discovery/settings/utils.py:17]: JWT is our preferred method of authentication.

**Comment by clintonb** [comment]
If your goal is simply to start a server, I don't think this commit is necessary. I'd prefer not to obfuscate Django's management commands behind Make targets unless absolutely necessary. My experience with devstack has shown me that hiding cruft behind paver leads to more cruft. We have a very simple app here. Your scripts should simply call `./manage.py runserver`. No need for Make.
